docs/xml/xml_data_best_practices.rst

```diff
@@ -0,0 +1,142 @@
+| PartyContest                           | pc                                    |
++----------------------------------------+---------------------------------------+
+| PartySelection                         | ps                                    |
++----------------------------------------+---------------------------------------+
+| Person                                 | per                                   |
++----------------------------------------+---------------------------------------+
+| PollingLocation                        | pl                                    |
++----------------------------------------+---------------------------------------+
+| RetentionContest                       | rc                                    |
++----------------------------------------+---------------------------------------+
+| Source                                 | src                                   |
++----------------------------------------+---------------------------------------+
+| State                                  | st                                    |
++----------------------------------------+---------------------------------------+
+| StreetSegment                          | ss                                    |
++----------------------------------------+---------------------------------------+
+
+
+.. _Hungarian-Style: http://en.wikipedia.org/wiki/Hungarian_notation
+
+File Structure
+--------------
+All XML files should be encoded UFT-8 and line breaks should be LF (\n) as opposed to CR LF (\r\n).
+
+For consistency across files and to aid human readability all lines should be indented two spaces and tabs should not
+be used.
+
+General Data Structure
+----------------------
+All data that are presented to end users of the data (i.e. contest names, referendum text, polling location names,
+street names, proper names), where possible, should be converted to Title Case to aid readability.
+
+All data should be trimmed to remove leading and trailing white space.
+
+Optional elements without values should be omitted from XML feed.
+
+Specific Data Types
+-------------------
+Elements with a data type of `xs:integer` must contain a valid whole number greater than zero.
+
+Elements with a data type of `xs:anyURI` should be entered as a fully qualified domain name
+(e.g. https://www.votinginfoproject.org/)
+
+Elements with a data type of `xs:dateTime` should be entered in `ISO-8601`_ format.
+
+Elements with a data type of `xs:boolean` should either have a value of `true` or `false`
+
+Elements with a data type of `xs:language` should contain a two character, lower-case, value corresponding to the
+`ISO 639`_ standard.
+
+Elements that have enumerations which include an `other` should have a corresponding value assigned to `OtherType` within
+the containing element.  For example:
+
+.. code-block:: xml
+   :linenos:
+
+ ``<BallotMeasureContest id="bm390616670907">
```

Comment:
The two backticks need to be struck here. The `code-block` takes care of the formatting. Just remember that the pre-formatted text's left edge has to line up with the `code-block` options (i.e. `:linenos:`). Let me know if this makes sense.